{"uuid": "9b079f54-a223-42d5-a46f-9a6de5eef26b", "children": ["0517061b-545c-440d-b2da-0822c5ee1f36"], "befores": [{"name": "xunit_setup_method_fixture_TestAddDevices", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not get focusPackageAndActivity. Original error: Error executing adbExec. Original error: 'Command 'C:\\\\Users\\\\Hello\\\\AppData\\\\Local\\\\Android\\\\Sdk\\\\platform-tools\\\\adb.exe -P 5037 -s R58N828LKMP shell dumpsys window displays' exited with code 1'; Command output: adb.exe: protocol fault (couldn't read status): connection reset\r\n\n\n", "trace": "  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\_pytest\\python.py\", line 841, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\_pytest\\python.py\", line 739, in _call_with_optional_argument\n    func(arg)\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\test_gzAndroidAuto.py\", line 277, in setup_method\n    master.wait_activity(\"com.glazero.android.SplashActivity\", 2, 2)\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\appium\\webdriver\\extensions\\android\\activities.py\", line 89, in wait_activity\n    WebDriverWait(self, timeout, interval).until(lambda d: d.current_activity == activity)\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 71, in until\n    value = method(self._driver)\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\appium\\webdriver\\extensions\\android\\activities.py\", line 89, in <lambda>\n    WebDriverWait(self, timeout, interval).until(lambda d: d.current_activity == activity)\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\appium\\webdriver\\extensions\\android\\activities.py\", line 73, in current_activity\n    return self.execute(Command.GET_CURRENT_ACTIVITY)['value']\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 31, in check_response\n    raise wde\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 26, in check_response\n    super().check_response(response)\n  File \"C:\\Users\\Hello\\PycharmProjects\\gz_ui_auto\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1659467433778, "stop": 1659467445357}], "start": 1659467433778, "stop": 1659467446143}